src/types/ethereum.ts

```diff
@@ -107,7 +107,7 @@ export const AddressDetailsEthereumTokens = extendCodec(
   AddressDetailsEthereumBasic,
   {},
   {
-    tokens: TokenDetailsERC20,
+    tokens: t.array(TokenDetailsERC20),
```

Comment:
This was the major culprit @dylanseago , the data comes back as an array in the blockbook response, but our types are setup to look for an object.

Reply:
Wow! Yep. That'd do it. Great work finding this.